jujugui/static/gui/src/app/components/deployment/summary/placement/test-placement.js

```diff
@@ -0,0 +1,91 @@
+/*
```

Comment:
Note for reviewer: this component is a duplicate of the existing placement tests.